crm_create_project/wizards/crm_create_project.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+from odoo import _, fields, models
+
+
+class CrmCreateProject(models.TransientModel):
+    _name = "crm.create.project"
+    _description = "Wizart to create Project from Lead/Opportunity"
+
+    project_name = fields.Char()
+    project_description = fields.Html()
+    lead_id = fields.Many2one("crm.lead")
+
+    def create_project(self):
+        project = (
+            self.env["project.project"]
+            .sudo()
+            .create(self._prepare_create_project_values())
+        )
+        self.lead_id.project_id = project
+        project.message_post(
+            body=_(
+                "Project created from lead/opportunity "
+                "<a href=# data-oe-model=crm.lead data-oe-id=%(lead)d>%(name)s</a>.",
+                lead=self.lead_id,
+                name=self.lead_id.name,
+            )
+        )
+        if self.lead_id:
+            self.lead_id.message_post(
+                body=_(
+                    "Project <a href=# data-oe-model=project.project "
+                    "data-oe-id=%(project)d>%(name)s</a> created.",
+                    project=project,
+                    name=project.display_name,
+                )
+            )
```

Comment:
Maybe you can use https://github.com/OCA/server-ux/tree/16.0/mail_message_destiny_link_template

```suggestion
            self.lead_id.message_post_with_view(
                'mail_message_destiny_link_template.message_destiny_link',
                values={'self': self.lead_id, 'destiny': project},
                subtype_id=self.env.ref('mail.mt_note').id,
            )
```